### PR TITLE
fix: budgie-extras

### LINKED ISSUE #2246: [BUG] budgie-extras
**Describe the bug**
Installing individual `budgie-applet*` packages may not install `budgie-extras-daemon` as dependency. It's not really a problem currently because both the `budgie-extras` metapackage and `budgie-extras-daemon` are dependencies of `ultramarine-flagship-filesystem`. However, this has the side effect of users of Flagship Edition not being able to remove any individual applets that they may not want. If you're going to the trouble of building separate packages for each applet, it makes sense to allow individual package removal.

**Suggested changes**
- `ultramarine-flagship-filesystem` - Change dependency on `budgie-extras` to recommends (won't change anything for builds because I believe `install_weak_deps=True` anyway)
- `ultramarine-flagship-filesystem` - Dependency on `budgie-applet-visualspace` (I think? this is actually the only truly essential package because it's used in UM Budgie default toolbar layout)
- `ultramarine-flagship-filesystem` - Remove dependency on `budgie-extras-daemon`
- `budgie-extras` - update specfile with which `budgie-applet*` packages depend on `budgie-extras-daemon`

**Affected (please complete the following information):**
 - UM Flagship Edition
 - `budgie-extras`, `ultramarine-flagship-filesystem`, `budgie-applet*`

-- 

If you're happy with these changes, I can open a PR on this repo for budgie-extras and a PR over on Ultramarine-Linux/packages for ultramarine-flagship-filesystem.

### PR DESCRIPTION
**Pull request by sadlerm4**
Closes #2246

### CODE REVIEW

**Comment by sadlerm4** [comment]
Do I need to bump the release number?

**Comment by madonuko** [comment]
Please do, thanks

File: anda/desktops/budgie/budgie-extras/budgie-extras.spec [hunk 1]
```diff
@@ -199,7 +199,7 @@ separately from the applet's menu
 
 %package -n	budgie-applet-takeabreak
 Requires:       budgie-extras-common
-Summary:m       A pomodoro-like applet, to make sure to take regular breaks from working
```
Comment (sadlerm4) [inline anda/desktops/budgie/budgie-extras/budgie-extras.spec]: m for... mado? lol

Comment (madonuko) [reply anda/desktops/budgie/budgie-extras/budgie-extras.spec]: lol

**Review by madonuko — APPROVED** [review]

**Pull request merged**